Errors found by vet, Println arg list ends with redundant newline

I got the following errors when I execute `make vet`.
```
go vet ./...
# github.com/kubesphere/kubekey/pkg/kubesphere
pkg/kubesphere/kubesphere.go:216:2: Println arg list ends with redundant newline
# github.com/kubesphere/kubekey/pkg/bootstrap
pkg/bootstrap/os.go:73:3: Println arg list ends with redundant newline
make: *** [vet] Error 2
```

It's fair enough, it's not necessary for a redundant newline.

--- a/.gitignore
+++ b/.gitignore
@@ -33,3 +33,6 @@
 *.un~
 Session.vim
 .netrwhist
+
+# Generate by makefile
+bin

--- a/pkg/bootstrap/os.go
+++ b/pkg/bootstrap/os.go
@@ -70,7 +70,7 @@ func InitOS(mgr *manager.Manager) error {
 			return err
 		}
 
-		fmt.Println("\nLocal images repository created successfully. Address: dockerhub.kubekey.local\n")
+		fmt.Print("\nLocal images repository created successfully. Address: dockerhub.kubekey.local\n")
 	}
 	return nil
 }

--- a/pkg/kubesphere/kubesphere.go
+++ b/pkg/kubesphere/kubesphere.go
@@ -213,7 +213,7 @@ func ResultNotes(incluster bool) error {
 		position = 1
 		notes    = "Please wait for the installation to complete: "
 	)
-	fmt.Println("\n")
+	fmt.Print("\n")
 	if incluster {
 		fmt.Println("Please wait for the installation to complete ...")
 	}

